GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -120,13 +120,31 @@ public string GVFSPostIndexChangedHookExecutableName
 
             public string MountExecutableName
             {
-                get { return "GVFS.Mount" + this.ExecutableExtension; }
+                get { return "gvfs.mount" + this.ExecutableExtension; }
             }
 
             public string GVFSUpgraderExecutableName
             {
                 get { return "GVFS.Upgrader" + this.ExecutableExtension;  }
             }
         }
+
+        public class UnderConstructionFlags
+        {
+            public UnderConstructionFlags(
+                bool supportsGVFSService = true,
+                bool supportsGVFSUpgrade = true,
+                bool supportsGVFSConfig = true,
+                bool supportsKernelLogs = true,
+                bool usesGitHooksLoader = false)
+            {
+            }
+
+            public bool SupportsGVFSService { get; }
+            public bool SupportsGVFSUpgrade { get; }
+            public bool SupportsGVFSConfig { get; }
+            public bool SupportsKernelLogs { get; }
+            public bool UsesGitHooksLoader { get; }
```

Comment:
I like that this is a lot more explicit.  I only thing that is weird is the `UsesGitHooksLoader` makes it look like that is something that is under construction when it is something that is being removed so maybe all the flags should have `true` defaults and `UsesGitHooksLoader` should be something like `NotUsingGitHooksLoader`?

Reply:
How about `RequiresDeprecatedGitHooksLoader`?

Reply:
Sounds good.  Thanks!